### PR TITLE
Feature/1738 readable date time

### PR DESCRIPTION
**Pull request by wmontgomery**
#1738 
image

### CODE REVIEW

**Comment by rathboma** [comment]
Hey Will,

Can we remove the icon and just make the title something like `${datetime} in unixtime`?

**Comment by wmontgomery** [comment]
You betcha.

**Comment by wmontgomery** [comment]
image

File: apps/studio/src/mixins/data_mutators.ts [hunk 1]
```diff
@@ -33,6 +33,14 @@ function emptyResult(value: any) {
   return null
 }
 
+function buildFormatterWithTooltip(cellValue: string, tooltip: string, icon?: string) {
+  if (!icon) {
+    return `<div class="cell-link-wrapper" title="${tooltip}">${cellValue}</div>`
```
Comment (rathboma) [inline apps/studio/src/mixins/data_mutators.ts:38]: These dynamic values need HTML escaping

File: apps/studio/src/mixins/data_mutators.ts [hunk 2]
```diff
@@ -53,22 +61,30 @@ export default {
...
           fkLink.onclick = (e) => params.fkOnClick(e, cell);
         })
-      }
+      } else if (
+          params?.isPK != null &&
+          !params.isPK &&
+          _.isInteger(Number(cellValue)) &&
+          Number(cellValue) <= 8640000000000000 // the date object doesn't go higher than said number https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#description
+        ) {
+        tooltip = `${new Date(Number(cellValue)).toISOString()} in unixtime`
```
Comment (rathboma) [inline apps/studio/src/mixins/data_mutators.ts:85]: What if this fails? We should error catch here and set a default tooltip

**Review by rathboma — CHANGES_REQUESTED** [review]
Error catching, html escaping, then done

File: apps/studio/src/mixins/data_mutators.ts [hunk 3]
```diff
@@ -79,11 +78,14 @@ export default {
...
-          Number(cellValue) <= 8640000000000000 // the date object doesn't go higher than said number https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#description
+          _.isInteger(Number(cellValue))
         ) {
-        tooltip = `${new Date(Number(cellValue)).toISOString()} in unixtime`
-        result = buildFormatterWithTooltip(cellValue, tooltip)
+        try {
+          tooltip = `${new Date(Number(cellValue)).toISOString()} in unixtime`
+          result = buildFormatterWithTooltip(cellValue, tooltip)
+        } catch (e) {
+          console.error(`${cellValue} cannot be converted to a date`)
```
Comment (rathboma) [inline apps/studio/src/mixins/data_mutators.ts:87]: You should log the error here too

**Review by rathboma — APPROVED** [review]

**Pull request merged**